azkaban-common/src/main/java/azkaban/imagemgmt/permission/PermissionManager.java

```diff
@@ -36,4 +36,11 @@
   public boolean hasPermission(final String imageTypeName, final String userId, final Type type)
       throws ImageMgmtException;
 
+  /**
+   * Checks if the user has permission to manage high priority flows.
+   * @param userId
+   * @return
+   * @throws ImageMgmtException
+   */
+  public boolean hasPermission(final String userId) throws ImageMgmtException;
```

Comment:
Method name should be hasHPFlowPermission

Reply:
I kept it same as there is already a method with same name.